controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -14,31 +14,7 @@
 import java.time.Duration;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static io.pravega.shared.MetricsNames.CREATE_STREAM;
-import static io.pravega.shared.MetricsNames.CREATE_STREAM_FAILED;
-import static io.pravega.shared.MetricsNames.CREATE_STREAM_LATENCY;
-import static io.pravega.shared.MetricsNames.DELETE_STREAM;
-import static io.pravega.shared.MetricsNames.DELETE_STREAM_FAILED;
-import static io.pravega.shared.MetricsNames.DELETE_STREAM_LATENCY;
-import static io.pravega.shared.MetricsNames.OPEN_TRANSACTIONS;
-import static io.pravega.shared.MetricsNames.RETENTION_FREQUENCY;
-import static io.pravega.shared.MetricsNames.SEAL_STREAM;
-import static io.pravega.shared.MetricsNames.SEAL_STREAM_FAILED;
-import static io.pravega.shared.MetricsNames.SEAL_STREAM_LATENCY;
-import static io.pravega.shared.MetricsNames.SEGMENTS_COUNT;
-import static io.pravega.shared.MetricsNames.SEGMENTS_MERGES;
-import static io.pravega.shared.MetricsNames.SEGMENTS_SPLITS;
-import static io.pravega.shared.MetricsNames.TRUNCATE_STREAM;
-import static io.pravega.shared.MetricsNames.TRUNCATE_STREAM_FAILED;
-import static io.pravega.shared.MetricsNames.TRUNCATE_STREAM_LATENCY;
-import static io.pravega.shared.MetricsNames.UPDATE_STREAM;
-import static io.pravega.shared.MetricsNames.UPDATE_STREAM_FAILED;
-import static io.pravega.shared.MetricsNames.UPDATE_STREAM_LATENCY;
-import static io.pravega.shared.MetricsNames.CREATE_KVTABLE_LATENCY;
-import static io.pravega.shared.MetricsNames.CREATE_KVTABLE;
-import static io.pravega.shared.MetricsNames.KVTABLE_SEGMENTS_COUNT;
-import static io.pravega.shared.MetricsNames.CREATE_KVTABLE_FAILED;
-import static io.pravega.shared.MetricsNames.globalMetricName;
+import static io.pravega.shared.MetricsNames.*;
```

Comment:
Please, use single line imports as it was before.

Reply:
This optimization is done by IntelliJ and is not done manually.

Reply:
fixed